helpers/collaborators.ts

```diff
@@ -0,0 +1,44 @@
+import * as Octokit from '@octokit/rest';
+import { exit } from './utils';
+import { Collaborator } from '../interfaces/collaborators';
+
+const getCollaborators = async ({
+  octokit,
+  owner,
+  repo,
+}: {
+  octokit: Octokit;
+  owner: string;
+  repo: string;
+}): Promise<Array<Collaborator>> => {
+  const projectCollaborators = await octokit
+    .paginate('GET /repos/:owner/:repo/collaborators', { owner, repo })
+    .catch((error) => {
+      exit('getCollaborators', error.message);
+    });
+
+  if (!Array.isArray(projectCollaborators)) {
```

Comment:
Because octokit _can_ return void values (even though they should probably end up in the catch...) the code becomes lots more defensively programmed (which probably isn't the worse thing in the world)